installer/Migration/V5_5_0/Migration.php

```diff
@@ -36,8 +36,11 @@ class Migration extends AbstractMigration
     public function up(): void
     {
         $this->insertI18nGroups();
+        $groups = ['claim', 'general'];
+        foreach ($groups as $group) {
+            $this->getLangStringHelper()->insertOrUpdateLangStrings($group);
+        }
         $this->getLangStringHelper()->deleteNonCustomizedLangStrings('claim');
```

Comment:
This is a critical issue. I fixed here https://github.com/orangehrm/orangehrm/pull/1711/commits/ac4cdb8e9c7d4b3582d70282c2272f59293c2fbe